articles/azure-large-instances/workloads/epic/register-the-ali-resource-provider.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Register the ALI source provider
+description: Explains how to register the ALI source provider.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+
+# Register the ALI source provider
```

Comment:
@ryellajo12 At some point this article fell out of the ToC - I don't remember if you told me to pull it or not. I do remember that it consists of a link from the Private Preview PDF to https://learn.microsoft.com/en-us/azure/baremetal-infrastructure/connect-baremetal-infrastructure?tabs=azure-cli. We can port this from BareMetal to ALI, but it's a risk for GA. Alternatively we could link to the existing BMI article from somewhere in the ALI content, with the caveat that it's BMI.

Reply:
@ianmsgit : This is same as the article that is already ported. https://github.com/MicrosoftDocs/azure-docs/pull/110785/files#diff-cc3f3396b8cd8ff2bb883ff619ded56af0ee856bd4a72ca0555f161f3e0c792e 

No need to migrate this again.It is already taken care of.

Reply:
Oh! So just delete then.

Fixed
To confirm in build

Reply:
Yes, Please